esmvaltool/recipes/hydrology/recipe_globwat.yml

```diff
@@ -0,0 +1,97 @@
+# ESMValTool
+# globwat.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the GlobWat hydrological model.
+    GlobWat is a Global Hydrological Model provided by FAO
+    http://www.fao.org/nr/water/aquamaps/
+
+    By default, PET is calculated by the De Bruin method (De Bruin et al. 2016).
+    However, users can choose the Langbein method by setting the langbein_pet
+    (Langbein et al. 1949) in the recipe to True.
+
+    More information about the De Bruin method can be found at:
+    https://doi.org/10.1175/JHM-D-15-0006.1 page 1376, equation 6.
+    More information about the Langbein method can be found at:
+    https://doi.org/10.3133/cir52 page 8, figure 1.
+    An example of using Langbein method can be found at:
+    https://doi.org/10.1080/02626667.2017.1332416 page 1472, equation 7.
+
+    A target file has been generated from one of the models sample files.
+    The target file can be found at:
+    https://github.com/babdollahi/globwat_target_grid
+
+    The rough cutout preprocessor can be used by users to process the data
+    for their desired region. The data will be processed at the global scale 
+    if the preprocessor in the recipe is commented.
+
+  authors:
+    - abdollahi_banafsheh
+    - alidoost_sarah
+
+  projects:
+    - ewatercycle
+
+  references:
+    - acknow_project
+
+preprocessors:
+    rough_cutout:
```

Comment:
I think something like 'area_selection' might be a more appropriate name for this preprocessor, if you agree.

Reply:
I agree and changed the name to “area_selection”